CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ the code was deployed.
 - "doorMissedFrequently" field to JSON data submitted by brave sensors that is sent to /api/heartbeat (CU-860rk8v2a)
```

Comment:
Can you please run the smoke tests and note that you did so in the PR description?

Reply:
Sure!